src/Jackett.Common/Services/ConfigurationService.cs

```diff
@@ -81,33 +81,44 @@ public void CreateOrMigrateSettings()
                     }
                     else
                     {
-                        PerformMigration();
+                        PerformMigration(oldDir);
```

Comment:
There's no need for this call, because if oldDir doesn't exist, this won't do anything in PerformMigration.

Reply:
Addressed in 1ade5585761a6e206580533ca99bed3d73d93625